build.sc

```diff
@@ -94,7 +96,9 @@ trait AWSSpec extends Cross.Module[String] with BaseModule {
     ivy"software.amazon.smithy:smithy-aws-traits:$smithyVersion",
     ivy"software.amazon.smithy:smithy-aws-cloudformation-traits:$smithyVersion",
     ivy"software.amazon.smithy:smithy-aws-iam-traits:$smithyVersion",
-    ivy"software.amazon.smithy:smithy-waiters:$smithyVersion"
+    ivy"software.amazon.smithy:smithy-waiters:$smithyVersion",
+    ivy"software.amazon.smithy:smithy-aws-endpoints:$smithyVersion",
+    ivy"software.amazon.smithy:smithy-aws-smoke-test-model:$smithyVersion"
```

Comment:
**question:** what are we doing about those? https://github.com/disneystreaming/aws-sdk-smithy-specs/issues/9#issuecomment-2383249207